bundles/org.openhab.binding.coolmasternet/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -7,6 +7,7 @@
 	<bridge-type id="controller">
 		<label>CoolMasterNet Controller</label>
 		<description>A CoolMasterNet Controller (connected to one or more HVAC systems)</description>
+		<semantic-equipment-tag>HVAC</semantic-equipment-tag>
```

Comment:
NetworkAppliance ?

the other thing in this file could be tagged with HVAC.

Reply:
Ok. Done.